rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json

```diff
@@ -0,0 +1,554 @@
+                                                        }
+                                                    ]
+                                                }
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            },
+            "baseline": {
+                "id": "ASHRAE229 1",
+                "ruleset_model_instances": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_open_schedule": "Required Building Schedule 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                                "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                                "terminals": [
+                                                    {
+                                                        "id": "Air Terminal",
+                                                        "is_supply_ducted": true,
+                                                        "type": "CONSTANT_AIR_VOLUME",
+                                                        "served_by_heating_ventilating_air_conditioning_system": "System 10",
+                                                        "heating_source": "ELECTRIC",
+                                                        "primary_airflow": 1000,
+                                                        "supply_design_heating_setpoint_temperature": 105
+                                                    }
+                                                ],
+                                                "design_thermostat_heating_setpoint": 65,
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
+                                                        "lighting_space_type": "STAIRWELL"
+                                                    }
+                                                ]
+                                            }
+                                        ],
+                                        "heating_ventilating_air_conditioning_systems": [
+                                            {
```

Comment:
This system isn't identified as any system. To be identified as system 10, it seems like ```cooling_system``` and ```cooling_system_type: "NONE"``` are missing. Please double-check this.

Reply:
I'm not sure what I should do here. It seems like all Systems 9 & 10 in the TCD Spreadsheet for section 19 leave the `cooling_system` and `cooling_system_type` blank the same as here. Is this issue for all systems 9 & 10?

Reply:
@jugonzal07 Any idea why the template for system 10 isn't being identified here?

Reply:
We discussed this thru emails. Please let me know if you have further questions. We can have a brief call and I'll show you how we can get this issue fixed.